emrun.py

```diff
@@ -637,12 +637,18 @@ def send_head(self):
     # gzipped file, instead of having the browser decompress it immediately,
     # then it can't use the suffix .gz when using emrun.
     # To work around, one can use the suffix .gzip instead.
```

Comment:
There are actually two conventions in use, one is to utilize `foo.datagz` and another to utilize `foo.data.gz`. The logic here is intentional in emrun to enable running content that has been authored with either convention.

The two conventions are there since some web servers (Apache or IIS, can't quite recall. I remember Nginx didn't have a problem here) are not that great at supporting two suffixes as extension, and e.g. attempting to set `Content-Type: application/wasm` to files with suffix `.wasm.gz` would fail, since they think the file has a suffix `.gz` only.